rd-cpp/src/rd_framework_cpp/src/main/scheduler/SingleThreadScheduler.cpp

```diff
@@ -16,6 +16,7 @@ SingleThreadScheduler::SingleThreadScheduler(Lifetime lifetime, std::string name
 		}
 		catch (std::exception const& e)
 		{
+			(void)e;
```

Comment:
You can instead just write `catch (std::exception const&)`
Same in next similar cases.

Reply:
That's true, but I've intentionally left exception value named and have explicitly suppressed unused variable warning to be able to set a breakpoint inside catch block and inspect exception value without recompilation. I don't know if it's possible with anonymous catch block.

Reply:
Ah, I see. I guess that it gets eliminated by compiler in release mode anyways.